sdk/basyx/tutorial/tutorial_create_simple_aas.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
+# This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+# See http://creativecommons.org/publicdomain/zero/1.0/ for more information.
+"""
+Tutorial for the creation of a simple Asset Administration Shell, containing an AssetInformation object and a Submodel
+reference using aas-core3.0-python
+"""
+
+# Import all type classes from the aas-core3.0-python SDK
+import aas_core3.types as aas_types
+
+# In this tutorial, you'll get a step-by-step guide on how to create an Asset Administration Shell (AAS) and all
+# required objects within. First, you need an AssetInformation object for which you want to create an AAS. After that,
+# an Asset Administration Shell can be created. Then, it's possible to add Submodels to the AAS. The Submodels can
+# contain SubmodelElements.
+
+# Step-by-Step Guide:
+# Step 1: create a simple Asset Administration Shell, containing AssetInformation object
+# Step 2: create a simple Submodel
+# Step 3: create a simple Property and add it to the Submodel
+
+
+############################################################################################
+# Step 1: Create a Simple Asset Administration Shell Containing an AssetInformation object #
+############################################################################################
+# Step 1.1: create the AssetInformation object
+asset_information = aas_types.AssetInformation(
+    asset_kind=aas_types.AssetKind.INSTANCE,
+    global_asset_id='http://acplt.org/Simple_Asset'
+)
+
+# Step 1.2: create the Asset Administration Shell
+identifier = 'https://acplt.org/Simple_AAS'
+aas = aas_types.AssetAdministrationShell(
+    id=identifier,  # set identifier
+    asset_information=asset_information,
+    submodels=[]
+)
+
+
+#############################################################
+# Step 2: Create a Simple Submodel Without SubmodelElements #
+#############################################################
+
+# Step 2.1: create the Submodel object
+identifier = 'https://acplt.org/Simple_Submodel'
+submodel = aas_types.Submodel(
+    id=identifier,
+    submodel_elements=[]
+)
+
+# Step 2.2: create a reference to that Submodel and add it to the Asset Administration Shell's `submodel` set
+submodel_reference = aas_types.Reference(
+    type=aas_types.ReferenceTypes.MODEL_REFERENCE,
+    keys=[aas_types.Key(
+        type=aas_types.KeyTypes.SUBMODEL,
+        value=identifier
+    )]
+)
+aas.submodels.append(submodel_reference)
```

Comment:
There's an issue with the `append` on a potential `None` object. 
As discussed, it might make sense to simply overwrite the `aas.submodels` list here in the tutorial, not to confuse readers. 

We might want to add a warning though: 
```
# Warning, this overwrites whatever is in the `aas.submodels` list. In your production code, it might make sense to check for already existing content. 
```

or something similar.